src/Shipping/PickupLocation.php

```diff
@@ -26,6 +26,7 @@ public function init() {
 		$this->title            = $this->get_option( 'title' );
 		$this->tax_status       = $this->get_option( 'tax_status' );
 		$this->cost             = $this->get_option( 'cost' );
+		$this->supports         = [ 'local-pickup' ];
```

Comment:
This should be documented, we currently don't have a dev documentation page for Local Pickup, but once we do, we should include this. Also this PR needs a dev note for this one.